catalog/spatial/csw/spatial-csw-endpoint/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -158,11 +158,19 @@
 
     <reference id="cswMetacardType" interface="ddf.catalog.data.MetacardType" filter="(name=csw:Record)"/>
 
+    <bean id="metacardTypes" class="ddf.catalog.util.impl.SortedServiceList"/>
+    <reference-list id="metacardTypeList"
+                    interface="ddf.catalog.data.MetacardType">
+        <reference-listener bind-method="bindPlugin" unbind-method="unbindPlugin"
+                            ref="metacardTypes"/>
+    </reference-list>
+
     <bean id="cswFilterFactory"
           class="org.codice.ddf.spatial.ogc.csw.catalog.endpoint.CswQueryFactory">
         <argument ref="filterBuilder"/>
         <argument ref="filterAdapter"/>
         <argument ref="cswMetacardType"/>
+        <argument ref="metacardTypes"/>
```

Comment:
This list ends up not being complete between restarts. I'm not too familiar with OSGi, but I don't think it matters if it's being passed in as an argument or property since they're all just references